instance/templates/instance/ansible/secret-keys.yml

```diff
@@ -14,9 +15,14 @@ NOTIFIER_LMS_SECRET_KEY: '{{ secret_generator.NOTIFIER_LMS_SECRET_KEY }}'
 PROGRAMS_SECRET_KEY: '{{ secret_generator.PROGRAMS_SECRET_KEY }}'
 ECOMMERCE_SOCIAL_AUTH_EDX_OIDC_KEY : '{{ secret_generator.ECOMMERCE_SOCIAL_AUTH_EDX_OIDC_KEY }}'
 ECOMMERCE_SOCIAL_AUTH_EDX_OIDC_SECRET : '{{ secret_generator.ECOMMERCE_SOCIAL_AUTH_EDX_OIDC_SECRET }}'
-ECOMMERCE_EDX_API_KEY: '{{ secret_generator.EDXAPP_EDX_API_KEY }}' # Same as the edxapp API key
 EDXAPP_ANALYTICS_API_KEY: '{{ secret_generator.EDXAPP_ANALYTICS_API_KEY }}'
 FORUM_API_KEY: '{{ secret_generator.FORUM_API_KEY }}'
-EDXAPP_COMMENTS_SERVICE_KEY: '{{ secret_generator.FORUM_API_KEY }}'
-NOTIFIER_COMMENT_SERVICE_API_KEY: '{{ secret_generator.NOTIFIER_COMMENT_SERVICE_API_KEY }}'
-NOTIFIER_USER_SERVICE_API_KEY: '{{ secret_generator.NOTIFIER_USER_SERVICE_API_KEY }}'
+
+# Keys that need to match other keys.  The variables will be filled in by Jinja2 template
+# processing in Ansible rather than by the Django template processor, hence the verbatim block.
+{% verbatim %}
+ECOMMERCE_EDX_API_KEY: '{{ EDXAPP_EDX_API_KEY }}'
```

Comment:
@smarnach So far we had been setting all the variables directly from the Django side, it avoid having to care about who is rendering what - why would this value from ansible differ from `secret_generator.EDXAPP_EDX_API_KEY` which is the value set above for `EDXAPP_EDX_API_KEY`? https://github.com/open-craft/opencraft/blob/e1825da5438a0cad47767c1743d4e0f506243620/instance/templates/instance/ansible/secret-keys.yml#L11